pkg/env/core.go

```diff
@@ -75,12 +75,15 @@ func (c *core) Logger() *logrus.Entry {
 }
 
 func (c *core) NewLiveConfigManager(ctx context.Context) (liveconfig.Manager, error) {
-	msiAuthorizer, err := c.NewMSIAuthorizer(c.Environment().ResourceManagerScope)
+	tokenizer, err := c.NewMSITokenCredential()
```

Comment:
super nit: Should it be `tokenizer` or `token`? I guess `token` is better.

Reply:
Fixed in updated commit